tests/integration/suite/daprd/serviceinvocation/grpc/fuzz.go

```diff
@@ -82,7 +82,7 @@ func (f *fuzzgrpc) Run(t *testing.T, ctx context.Context) {
 	f.daprd2.WaitUntilRunning(t, ctx)
 
 	pt := parallel.New(t)
-	for i := 0; i < len(f.methods); i++ {
+	for i := range len(f.methods) {
```

Comment:
```suggestion
	for i := range f.methods {
```
Just a nit: We don't have to get the length when using range because `i` is the index. I'm only bringing it up for consistency's sake because we had it like this in the other linter PR.

Reply:
TIL ^ - I like that it returns just the index rather than the value in the slice too